admin_guide/high_availability.adoc

```diff
@@ -580,6 +585,34 @@ the list of VIPs.  The vrrp-ids may have values in the range 1..255.
 
 When there are multiple ipfailover deployment configuration care must be taken to specify `--vrrp-id-offset` so that there is room to increase the number of VIPS in the deployment configuration and none of the vrrp-id ranges overlap.
 
+[[admin-guide-high-availability-configuring-more-than-254]]
+=== Configuring IPfailover for more than 254 addresses
+
+By default, there is a limit of 254 VRRP IDs for any ethernet broadcast domain. You can use
+the `--virtual-ip-groups` option
+xref:..//admin_guide/high_availability.adoc#configuring-ip-failover[configuring
+IP failover] to determine the number of VRRP IDs that are used for IPfailover.
+
+----
+# oc adm ipfailover <ipfailover_name> --create \
+    ...
+    --virtual-ip-groups=<number_of_ipfailover_groups>
+----
+
+[NOTE]
+====
+If the number of groups set by the `--virtual-ip-groups` option is fewer than
+the number of IP addresses set to fail over, the group will contain more than
+one IP address, and all of the addresses will move together.
```

Comment:
Is "move together" the best way to describe the negative effects? Is there a better or more descriptive way to say it?

Reply:
Move as a single unit?